tests/Utilities/MakeWithRandomValues.hpp

```diff
@@ -70,6 +70,53 @@ struct FillWithRandomValuesImpl<Variables<tmpl::list<Tags...>>,
 /// \endcond
 }  // namespace TestHelpers_detail
 
+/// \ingroup TestingFrameworkGroup
+/// \brief A uniform distribution functor for which the behavior can be easily
+/// customized
+///
+/// \details The two constructors take either a low and high bound arguments or
+/// a similar 2-component array , which can be passed in in any type from which
+/// the generated type can be constructed. The default generation function
+/// operates as would be expected for a double
+template <typename T>
+struct uniform_custom_distribution {
+  using result_type = T;
```

Comment:
`value_type` seems more consistent with the STL, why `result_type`?

Reply:
needs to be that way to interact properly with anything that uses the distribution as a distribution. Patterned after std::uniform_real_distribution [cpp reference - uniform_real_distribution](https://en.cppreference.com/w/cpp/numeric/random/uniform_real_distribution)